cdc/cdc_extension.hh

```diff
@@ -20,6 +20,7 @@
 
 #pragma once
 
+#include "schema.hh"
```

Comment:
Please add this in patch that introduces the dependency on schema_extension. Unless it's not one of the patches of this PR.